device-connectors/src/testflinger_device_connectors/devices/oem_autoinstall/oem_autoinstall.py

```diff
@@ -76,6 +76,7 @@ def provision(self):
 
             if not default_user_data.exists():
                 raise ProvisioningError("Default user-data file not found")
+            subprocess.run(["mkdir", "-p", ATTACHMENTS_PROV_DIR])
```

Comment:
I think it is better to use the built-in function which handles the errors (`raise OSError`).

```python
os.makedirs(ATTACHMENTS_PROV_DIR, exist_ok=True)
```

Reply:
thanks for the suggestion, just used the built-in function.